apps/tasks/src/App.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <Molgenis id="__top" v-model="session">
+    <div v-if="session && session.email == 'admin'" class="card">
+      <div class="card-header">
+        <ul class="nav nav-tabs card-header-tabs">
+          <li
+            class="nav-item"
+            v-for="(label, key) in {
+              scripts: 'Scripts',
+              jobs: 'Jobs',
+            }"
+            :key="key"
+          >
+            <router-link
+              class="nav-link"
+              :class="{
+                active: selected == key,
+                'text-danger': key === 'settings',
+              }"
+              :to="key"
+              >{{ label }}
+            </router-link>
+          </li>
+        </ul>
+      </div>
+      <div class="card-body">
+        <router-view :session="session" />
+      </div>
+    </div>
+    <MessageError v-else>
```

Comment:
this is briefly show and the removed when the user has the rights to view. Makes the app feel a bit buggy. Suggest te wait for response before showing error